cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -71,7 +71,8 @@ def full_test_vm(setup_provider_modscope, provider, full_template_modscope, requ
 def new_snapshot(test_vm, has_name=True, memory=False):
     return Vm.Snapshot(
         name="snpshot_{}".format(fauxfactory.gen_alphanumeric(8)) if has_name else None,
-        description="snapshot", memory=memory, parent_vm=test_vm)
+        description="snapshot_{}".format(fauxfactory.gen_alphanumeric(8)),
```

Comment:
**QUESTION:** Is this the same on all streams?

Reply:
It _should_ be. PRT says it's good with 5.7 and 5.8. This is the behaviour seen with RHV provider.

After all, it's just a text change in the description of a snapshot.